okta_connector.py

```diff
@@ -978,6 +1110,11 @@ def initialize(self):
             return self.set_status(phantom.APP_ERROR, "Error occurred while getting the Phantom server's Python major version.")
 
         self._state = self.load_state()
+        if not isinstance(self._state, dict):
+            self.debug_print("Resetting the state file with the default format")
+            self._state = {"app_version": self.get_app_json().get("app_version")}
+            return self.set_status(phantom.APP_ERROR, OKTA_STATE_FILE_CORRUPT_ERR)
```

Comment:
return statement can be removed as no data is stored in the state file except app_version

Reply:
Fixed